### PR TITLE
Iteration 8 and 9 completed

### PR DESCRIPTION
**Pull request by JoelLindow**
All Hail Tom! Iterations completed.

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (mdevoe12) [inline spec/models/user_spec.rb]: Can we be sure to add model tests in the next iteration?

Comment (JoelLindow) [inline spec/models/user_spec.rb]: Sure thing Matt. Whenever you want to get that started I'll be happy to support that winning choice you're making for our team. Go for it, chief!

Comment (mdevoe12) [inline spec/models/user_spec.rb]: 👍

Comment (JoelLindow) [inline spec/models/user_spec.rb]: 🥇 Big Guy

**Pull request merged**